airbyte-integrations/connectors/source-salesforce-singer/metadata.yaml

```diff
@@ -9,11 +9,11 @@ data:
   name: Salesforce (Singer)
```

Comment:
Interesting whats the diff between salesforce and salesforce singer. Keep a not. Not related to this PR.